call-for-tutorials.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: Call for Tutorials
+permalink: /call-for-tutorials
+---
+
+
+The **{{ site.inline-logo }}** is inviting proposals for tutorials on core machine learning topics or emerging areas of significance for machine learning. Tutorials should be self-contained, covering necessary background material, and demonstrate their importance to the ML in PL community.
+
+Only research institutions are eligible to submit tutorial proposals. Tutorials should reflect the wider machine learning research, not solely focus on presenters or their organizations. They should provide a balanced overview of the research area, including both known results and open problems.
+
```

Comment:
Mozna dodac to zdanie "Tutorials with a clear marketing purpose will not be accepted."/przekopiowac tekst z Paperforma 2024.

Reply:
Done